SR2021/2020-02-28-srawn-03.md

```diff
@@ -0,0 +1,39 @@
+The Student Robotics (Almost) Weekly Newsletter is a (roughly) fortnightly newsletter which summarises the goings‐on across Student Robotics. **Emboldened** items are ones for which additional help has been requested.
+
+# Headlines
+
+### Mentoring
+
+**There are a number of teams who need mentors to help them with this year's game.** No experience with mentoring or the simulator required, but experience with SR is definitely useful!
+
+If you're interested, please let the competition team know!
+
+### Module III League
+
+We announced Module III at the league session, and teams have 6 weeks to react until the first league session. See [the announcement](studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.
```

Comment:
Maybe remove "first"? It's a little confusing as this won't be the first league session -- it's going to be the third (overall).
```suggestion
We announced Module Ⅲ at the last league session, and teams have 6 weeks to react until the next league session. See [the announcement](studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.
```

Reply:
Yeah, absolutely agree. Looks like this content was copy-pasted from a previous issue when it was submitted.